circuits/authOnChainSmt.circom

```diff
@@ -0,0 +1,5 @@
+pragma circom 2.0.0;
+
+include "lib/authenticationOnChainSmt.circom";
+
+component main {public [userID,challenge,userState]} = VerifyAuthenticationOnChainSmt(32,32);
```

Comment:
In on-chain SMT version, `userState` should not be visible as public input, or in this case, it is a global state?

Reply:
fixed